### ISSUE TITLE
Bookmark sync does not work after updating to 17.2 

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#5036]
So far I was unable to reproduce this issue but we had several reports of bookmark sync not working. 

https://bugzilla.mozilla.org/show_bug.cgi?id=1554434
https://bugzilla.mozilla.org/show_bug.cgi?id=1554413

I tried updating from: 
- 16.0 to 17.2
- 16.2 to 17.2
- 17.0 to 17.2
- 17.1 to 17.2
- 16.2 to 17.0 to 17.1 to 17.2

None of the tested scenarios reproduced the issue.

**Comment by justindarc** [comment]
The issue has been identified:
```
2019-05-28 17:28:08.776 [Warning] [Profile.swift:265] init(localName:syncDelegate:clear:) > [RUST][sync15::sync_multiple] Sync of bookmarks failed! Error(

Store error: Error synchronizing: JSON error: invalid type: string "1453040847276", expected i64)
```

This has already been fixed in v0.29.0 of mozilla/application-services: https://github.com/mozilla/application-services/compare/v0.28.1...v0.29.0

**Comment by justindarc** [comment]
I believe we already fixed this when we updated application-services. If more issues get reported we can open a new issue.
